test_files/materials_test.py

```diff
@@ -98,6 +98,15 @@ def _set_test_mcprep_texturepack_path(self, reset: bool = False):
             raise Exception("Failed to set test texturepack path")
         bpy.context.scene.mcprep_texturepack_path = path
 
+    def _count_missing_files(self) -> int:
+        missing = 0
+        for img in bpy.data.images:
+            if not os.path.isfile(img.filepath):
+                missing += 1
+            elif img.channels == 0:
```

Comment:
The check for channels was a good proxy to see if data is loaded or not, found anecdotally that it's set to zero if the image failed to actually load (which could be true even if the above isfile passes, due to the blender native issue of failing to load GPU textures)

Reply:
I think we should add a small comment as well explaining why checking channels is needed, in case future developers come across this